flash_learning/main/routes.py

```diff
@@ -23,7 +23,7 @@ def login():
     """"Student login page."""
 
     if current_user.is_authenticated:
-        return redirect(url_for("index"))
+        return redirect(url_for("main.index"))
```

Comment:
I think we need the full path to the index method, but I will try it out first to be sure.